package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "build": "node ./scripts/build",
     "prepublish": "node ./scripts/prepublish",
-    "doc-build": "./scripts/yuidoc",
+    "doc-build": "./scripts/build-doc",
```

Comment:
It's dumb but it would be nice if the file `build-doc` was actually named `doc-build`.